common/app/common/configuration.scala

```diff
@@ -210,13 +210,24 @@ class GuardianConfiguration extends GuLogging {
 
     lazy val key: Option[String] = configuration.getStringProperty("content.api.key")
     lazy val timeout: FiniteDuration =
-      Duration.create(configuration.getIntegerProperty("content.api.timeout.millis").getOrElse(2000), MILLISECONDS)
+      Duration.create(
+        configuration
+          .getIntegerProperty("content.api.timeout.millis")
+          .getOrElse(2000L)
+          .asInstanceOf[Number]
+          .longValue(),
```

Comment:
What does this do? Convert the optional integer from configuration to a `Long`? Why use `Long` over `Int`, if we never expect these settings to be close to the upper limit of `Int`?

Reply:
Yeah, that's not very beautiful... Unfortunately we have to make this conversion because [`Duration.create` expects a `Long`](https://www.scala-lang.org/api/current/scala/concurrent/duration/Duration$.html) and `timeout` is a `FiniteDuration` object with this constructor: [`FiniteDuration(length: Long, unit: TimeUnit)`](https://www.scala-lang.org/api/current/scala/concurrent/duration/FiniteDuration.html) 🤷‍♀️